addons/wp/stringtable.xml

```diff
@@ -516,6 +516,12 @@
             <Polish>Obszar na którym grupa będzie szukać przeciwników.\nDomyślnie moduł wyszukuje tylko wrogich graczy.</Polish>
             <Russian>Расстояние на котором группа охоты будет искать врагов.\nПо умолчанию этот модуль нацелен только на вражеских игроков.</Russian>
         </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskHunt_TryUGLFlare_DisplayName">
+            <English>Use Flare</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskHunt_TryUGLFlare_Tooltip">
+            <English>Should the Units use Flare on Contact.</English>

```

Comment:
Sets how units use flares when hunting for enemies.

Reply:
```suggestion
            <English>Sets how units use flares when hunting for enemies.</English>
```